release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -245,6 +245,11 @@ With this update, you can provision IPv6 address spoke clusters from dual-stack
 
 With this release, you can pre-cache your application workload images before upgrading your applications on {sno} clusters with {cgu-operator-full}. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc#talm-prechache-user-specified-images-concept_ztp-talm[Pre-caching user-specified images with TALM on single-node OpenShift clusters].
 
+[id="ocp-4-14-ztp-siteconfig-disk-cleaning"]
+==== Disk cleaning option through SiteConfig and GitOps ZTP
+
+With this release, you can enable removing the partitioning table before installation with `automatedCleaningMode` in the `SiteConfig` CR. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc#ztp-deploying-a-site_ztp-deploying-far-edge-sites[Deploying a managed cluster with SiteConfig and GitOps ZTP]
```

Comment:
> enable removing

This sounds odd to me. 
remove the partitioning table
allow the removal of the partitioning table